### PR TITLE
Update changelog for oso 0.11.1.

### PR DESCRIPTION
**Pull request by dhatch**

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ export JS_DIR := $(abspath ../languages/js)
 
 rwildcard=$(foreach d,$(wildcard $(1:=/*)),$(call rwildcard,$d,$2) $(filter $(subst *,%,$2),$d))
 
-JAVA_PACKAGE_JAR_PATH := $(JAVA_DIR)/target/oso-0.11.0.jar
+JAVA_PACKAGE_JAR_PATH := $(JAVA_DIR)/target/oso-0.11.1.jar
```
Comment (samscott89) [inline docs/Makefile:19]: Need to do the same in new-docs.

Comment (dhatch) [reply docs/Makefile:19]: fixed

**Comment by dhatch** [comment]
@samscott89 fixed new docs jar path, could you look again?

**Review by samscott89 — APPROVED** [review]
LGTM

**Pull request merged**